### ISSUE TITLE
Manifest error

### ISSUE THREAD

**Issue by GranDoxin** [Andrews54757/FastStream#187]
Hello! The extension is not installed in any chromium browser - it gives a manifest error. What can be wrong?

**Comment by Andrews54757** [comment]
Did you follow the instructions to install it? Please describe what you did to install it and what browser you are using.

**Comment by GranDoxin** [comment]
I downloaded the FastStream-main zip archive and unzipped it. In the browser's extension settings, I enabled developer mode, clicked the "load unpacked extension" button, and specified the path to this folder. Gives an error message.
I tried installing from the Google store - everything worked. However, in one of the browsers I cannot install from the store because it is the Сomodo Dragon - Google does not allow installation from the store in it. So I did the installation manually.
I also tried to manually install it in several other Chromium browsers - the same error occurred everywhere. 
![1](https://github.com/user-attachments/assets/84df282d-1458-4a70-aef2-5ef8b384227b)
![2](https://github.com/user-attachments/assets/fda40824-5da1-461e-8c0b-e88a3cdd81d3)
